api/http/proxy/local.go

```diff
@@ -1,18 +1,18 @@
 package proxy
 
-// unixSocketHandler represents a handler to proxy HTTP requests via a unix:// socket
```

Comment:
A fix has been introduced in this file via https://github.com/portainer/portainer/pull/1192

Please make sure it's implemented here as well.

Reply:
Git saved the day